index.js

```diff
@@ -67,15 +67,29 @@ var createLogger = require('./src/logger.js');
  *  Reporting
  */
 
-// TODO: Update this documentation
 /**
- * The entry point for initializing the Error Reporting Middleware. This
- * function will invoke configuration gathering and attempt to create a API
- * client which will send errors to the Error Reporting Service. Invocation of
- * this function will also return an interface which can be used manually via
- * the `report` function property, with hapi via the `hapi` object property or
- * with express via the `express` function property.
- * @function initConfiguration
+ * <p class="notice">
+ *   **This is not an official Google product.** This module is experimental
+ *   and may not be ready for use.  This module uses APIs that may be
+ *   undocumented and are subject to change without notice.
+ * </p>
+ *
+ * This module provides Stackdriver Error Reporting support for Node.js
+ * applications.
+ * [Stackdriver Error Reporting](https://cloud.google.com/error-reporting/) is
+ * a feature of Google Cloud Platform that allows in-depth monitoring and
+ * viewing of errors reported by applications running in almost any environment.
+ *
+ * This module initializes the Error Reporting Middleware and provides an object
+ * that can be used manually through the `report` function property, with Hapi
+ * via the `hapi` object property or with Express via the `express` function
```

Comment:
This description of the interface can be shortened if there is a way to connect this constructor with the `Errors` type defined above. Does the `@alias` below do that?